Tests/Fixtures/TranslatableItem.php

```diff
@@ -30,6 +30,11 @@ public function __construct($id = null, $translations = array())
         $this->currentTranslations = $translations;
     }
 
+    public function isPrimaryKeyNull(): bool
+    {
+        return false;
```

Comment:
🔧 Please add the proper check.
```suggestion
        return null === $this->getId();
```